recipe/meta.yaml

```diff
@@ -19,14 +19,16 @@ build:
 
 requirements:
   build:
-    - toolchain
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - python
     - setuptools
-    - numpy 1.11.*
+    - numpy
```

Comment:
@luispedro, do we need `numpy` version `1.11` as a minimum or is there some other minimum requirement for `numpy`?